lms/djangoapps/verify_student/tests/test_models.py

```diff
@@ -198,61 +169,36 @@ def test_name_freezing(self):
 
         self.assertEqual(u"Clyde \u01B4", attempt.name)
 
-    def create_and_submit(self):
-        """Helper method to create a generic submission and send it."""
-        user = UserFactory.create()
-        attempt = SoftwareSecurePhotoVerification(user=user)
-        user.profile.name = u"Rust\u01B4"
-
-        attempt.upload_face_image("Just pretend this is image data")
-        attempt.upload_photo_id_image("Hey, we're a photo ID")
-        attempt.mark_ready()
-        attempt.submit()
-
-        return attempt
-
     def test_submissions(self):
         """Test that we set our status correctly after a submission."""
         # Basic case, things go well.
-        attempt = self.create_and_submit()
-        self.assertEqual(attempt.status, "submitted")
+        attempt = self.create_upload_and_submit_attempt()
+        self.assertEqual(attempt.status, PhotoVerification.STATUS.submitted)
 
         # We post, but Software Secure doesn't like what we send for some reason
-        with patch('lms.djangoapps.verify_student.models.requests.post', new=mock_software_secure_post_error):
-            attempt = self.create_and_submit()
-            self.assertEqual(attempt.status, "must_retry")
+        with patch('lms.djangoapps.verify_student.tasks.requests.post', new=mock_software_secure_post_error):
+            attempt = self.create_upload_and_submit_attempt()
+            self.assertEqual(attempt.status, PhotoVerification.STATUS.must_retry)
 
         # We try to post, but run into an error (in this case a network connection error)
-        with patch('lms.djangoapps.verify_student.models.requests.post', new=mock_software_secure_post_unavailable):
```

Comment:
This part of the test verifies the logs were hit -- as this code is not in `tasks.py`, I have moved this part of the test in `test_tasks.py`